virtual_rainforest/data_variables.toml

```diff
@@ -477,14 +477,6 @@ initialised_by = "external"
 updated_by = "soil"
 used_by = [ "abiotic", "soil"]
 
-[[variable]]
-name = "percent_clay"
-description = "Clay as a percentage of total soil"
-unit = "%"
-initialised_by = "external"
-updated_by = ""
-used_by = [ "abiotic", "soil"]
```

Comment:
At the moment this is only indirectly used by the `hydrology` model but I guess we could make it more explicit in the future. Thus, would it be worth already adding the hydrology here? If we do it as implemented, we should probably remove the `abiotic` one here for now.

Reply:
Oh right I had not realised that, shall we chat about this tomorrow?

Reply:
Switched to using `clay_fraction` from using `clay_percentage` as it felt like a less confusing unit